src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,127 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    private static final String COLUMN_ID = "id";
+    private static final String COLUMN_NAME = "name";
+    private static final String LICENCE = "licence_number";
+    private static final String STATEMENT_UPDATE = "UPDATE drivers "
+            + "SET " + COLUMN_NAME + " = ?, " + LICENCE + " = ? "
+            + "WHERE id = ? " + "AND is_deleted IS FALSE;";
```

Comment:
why not use COLUMN_ID constant here, as you've done with COLUMN_NAME and LICENCE above? Let's make your code consistent and either use constants everywhere or don't use them at all in your queries